Sources_v3/StreamChatUI/ChatChannel/MessageComposerSendButton.swift

```diff
@@ -0,0 +1,53 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+open class MessageComposerSendButton: ChatSquareButton {
+    // MARK: Underlying types
+    
+    public enum Mode {
+        case new, edit
+    }
+    
+    // MARK: - Properties
+    
+    public var mode: Mode = .new {
+        didSet {
+            updateContent()
+        }
+    }
+    
+    // MARK: - Overrides
+    
+    override open var isEnabled: Bool {
+        didSet {
+            transform = isEnabled && mode == .new ? CGAffineTransform(rotationAngle: -CGFloat.pi / 2.0) : .identity
+        }
+    }
+    
+    override open func setUpLayout() {
+        guard let size = defaultIntrinsicContentSize else { return }
+        
+        heightAnchor.constraint(equalToConstant: size.height).isActive = true
+        widthAnchor.constraint(equalToConstant: size.width).isActive = true
+    }
+    
+    override open func updateContent() {
+        switch mode {
+        case .new:
+            let normalStateImage = UIImage(named: "sendMessageArrow", in: .streamChatUI)
+            setImage(normalStateImage, for: .normal)
```

Comment:
The normal state icons should respect the `tintColor`. Try to change the `tintColor` in `SceneDelegate.swift` to a different color and you'll see it doesn't work properly. The "blue" should be red:

<img width="346" alt="Screen Shot 2020-12-08 at 10 20 10" src="https://user-images.githubusercontent.com/6388510/101464458-f53b6c80-393e-11eb-9de2-f1a85a24a3b1.png">

Reply:
As we discussed, there is a thing with different scenes so added manual `tintColor` set for `inputAccessoryViewController`.
Will also mention that in `Notion`.